Add endpoints to get scopes and resources associated to a given permission

ntm97 opened the pull request:
Similar to what was described in #101, I was trying to obtain the list of scopes associated with a scope-based permission. However, the endpoint `<client-id>/authz/resource-server/permission/scope/<permission-id>` only returns objects like this (I am running keycloak 11.0.2):
```
{
  decisionStrategy: ...,
  id: ...,
  logic: ...,
  name: ...,
  type: ...
}
```

Since the keycloak web UI uses the endpoints `<client-id>/authz/resource-server/policy/<permission-id>/scopes` and `<client-id>/authz/resource-server/policy/<permission-id>/resources` to fetch and display the scopes and resources associated with a permission, I figured it would be a good idea to add them to the admin client.

jonkoops commented:
@edewit Are the AuthZ endpoints something we want to add to the admin client? It looks like there is no documentation available for this as far as I can tell.

edewit commented:
@jonkoops when it's used in the admin console we will need to add it here as well

edewit commented:
@ntm97 could you add a test for these?

jonkoops commented:
Sure if we need this, would be nice to have a link to some of the documentation though.

ntm97 commented:
@edewit I just added test for the endpoints I added, and some of the other authz endpoints too (I did not find any tests for them).

Also I ran into the problem that the `ResourceRepresentation` type has an `id` property, but keycloak always return authorization resources with `_id` not `ìd`. I am not sure why keycloak chooses to use `_id`, as far as I can tell resources are the only case where that happens.

Currently I just added `@ts-ignore` when I had to access the `id` of a resource in the tests, but we probably should just rename the `id` property of the `ResourceRepresentation` to `_id`?

edewit commented:
yeah that is weird @ntm97 see it's done here: https://github.com/keycloak/keycloak/blob/fdc0943a92f75cd55192b40931255f480bdf1133/core/src/main/java/org/keycloak/representations/idm/authorization/ResourceRepresentation.java#L43
I've went and changed that in the `ResourceRepresentation` and fixed some small formatting. Thanks again for this, great stuff

edewit commented:
@ntm97 it's released in version [16.0.0-dev.20](https://www.npmjs.com/package/@keycloak/keycloak-admin-client/v/16.0.0-dev.20)
